### ISSUE TITLE
On sensor detail screen, no time or value if cursor is outside graph - nit

### ISSUE THREAD

**Issue by DG12** [ruuvi/com.ruuvi.station.webui#138]
Then the cursor is outside the graph no time or value is should.
It should be the most recent time/value
<img width="717" alt="ruuvistation time--" src="https://user-images.githubusercontent.com/2365961/154315358-8add0fcd-2e65-410e-a3ce-4e692636fa53.png">

**Comment by laurijamsa** [comment]
The latest data point is already shown on above the graph, so this would be just duplicated data and therefore not necessarily needed. We could also hide the "Time: -- Movement Counter: --" completely if user isn't hovering on the graph. 

Another thing related to this is that when I tested this, I noticed that the latest data (-0,05C) is different than the latest data on the graph (-0,08C). Let's see if we could fix that.

![image](https://user-images.githubusercontent.com/1311912/154418044-1c26709b-19b5-4204-b7c4-ceb82ad50508.png)
